### PR TITLE
Update packages to build on Xcode13

### PR DESCRIPTION
**Pull request by muukii**
### Checklist
- [x] I've added at least one test that validates that my change is working, if appropriate
- [x] I've followed the code style of the rest of the project
- [x] I've read the [Contribution Guidelines](CONTRIBUTING.md)
- [x] I've updated the documentation if necessary

### CODE REVIEW

File: Sources/Rugby/Common/XcodeProj/Scheme/XcodeProj+RemoveSchemes.swift [hunk 1]
```diff
@@ -30,7 +30,7 @@ extension XcodeProj {
             }
 
             scheme.buildAction?.buildActionEntries.removeAll {
-                !remainingUUIDs.contains($0.buildableReference.blueprintIdentifier)
+                !remainingUUIDs.contains($0.buildableReference.blueprintIdentifier!)
```
Comment (muukii) [inline Sources/Rugby/Common/XcodeProj/Scheme/XcodeProj+RemoveSchemes.swift:33]: it's changed by updating the package.
I'm not sure if it's safe.
@swiftyfinch

File: Sources/Rugby/Common/XcodeProj/Scheme/XcodeProj+RemoveSchemes.swift [hunk 2]
```diff
@@ -21,7 +21,7 @@ extension XcodeProj {
 
         let customSchemesForRemove = (sharedData?.schemes ?? []).filter { scheme in
             scheme.testAction?.testables.removeAll {
-                !remainingUUIDs.contains($0.buildableReference.blueprintIdentifier)
+                !remainingUUIDs.contains($0.buildableReference.blueprintIdentifier!)
```
Comment (muukii) [inline Sources/Rugby/Common/XcodeProj/Scheme/XcodeProj+RemoveSchemes.swift:24]: https://github.com/swiftyfinch/Rugby/pull/47/files#r715469988

File: Sources/Rugby/Common/XcodeProj/Scheme/XcodeProj+RemoveSchemes.swift (hunk 1 above)
Comment (swiftyfinch) [reply Sources/Rugby/Common/XcodeProj/Scheme/XcodeProj+RemoveSchemes.swift:33]: Hey! Let's just unwrap it and skip this if nil

File: Sources/Rugby/Common/XcodeProj/Scheme/XcodeProj+RemoveSchemes.swift (hunk 2 above)
Comment (swiftyfinch) [reply Sources/Rugby/Common/XcodeProj/Scheme/XcodeProj+RemoveSchemes.swift:24]: The same here

**Comment by muukii** [comment]
@swiftyfinch 
I've fixed where you pointed out.
would you review this again?

**Comment by swiftyfinch** [comment]
@muukii Everything is OK.
Thanks for your contribution! 🏈

**Pull request merged**